src/View/Helper/FormHelper.php

```diff
@@ -601,12 +622,19 @@ public function secure(array $fields = [], array $secureAttributes = [])
             'value' => $tokenData['unlocked'],
         ]);
         $out .= $this->hidden('_Token.unlocked', $tokenUnlocked);
+        if (!empty($tokenData['optional'])) {
+            $tokenOptional = array_merge($secureAttributes, [
```

Comment:
if those are all string based array keys (assoc. array), it is better to use + operator for joining

    $attributes = ['value' => $tokenData['optional'] + $secureAttributes;